addon/services/poll.js

```diff
@@ -35,7 +35,9 @@ export default Service.extend({
   },
 
   stopPoll(handle) {
-    clearInterval(handle);
+    if(handle) {
```

Comment:
should we also check for `typeof clearInterval` here?

also a small style note. would you add a space between `if` and the opening paren, so it's `if (handle)`?